tests/utils/eol-conversion.spec.ts

```diff
@@ -64,7 +64,6 @@ describe('detectEncoding() function', function() {
 		const sampleBinary = [
 			'ext2fs/build/Release/bindings.node',
 			'drivelist/build/Release/drivelist.node',
-			'resin-cli-visuals/node_modules/drivelist/build/Release/drivelist.node',
```

Comment:
This file no longer exists because it's been deduped with the top-level drivelist